parsl/serialize/facade.py

```diff
@@ -1,22 +1,34 @@
-from parsl.serialize.concretes import *  # noqa: F403,F401
-from parsl.serialize.base import METHODS_MAP_DATA, METHODS_MAP_CODE
+from parsl.serialize.base import SerializerBase
 import logging
 
-from typing import Any, List, Union
+from typing import Any, Dict, List, Union
 
 logger = logging.getLogger(__name__)
 
+import parsl.serialize.concretes as concretes
 
```

Comment:
Did flake8 not complain?  I'm accustomed to seeing imports first (often organized by "closeness" to the project), before code lines.